src/System.Data.SqlClient/src/System/Data/SqlClient/SqlStream.cs

```diff
@@ -469,27 +469,27 @@ override public long Seek(long offset, SeekOrigin origin)
 
             if (null == _cachedBytes)
             {
-                throw ADP.StreamClosed(ADP.Read);
+                throw ADP.StreamClosed();
```

Comment:
I deliberately changed this from the previously incorrect "Read" to the correct "Seek" (via `[CallerMemberName]`).